src/email-sender.js

```diff
@@ -0,0 +1,38 @@
+var nodemailer = require('nodemailer');
+
+/*************************************************************************
+*                             HOW TO USE                                 *
+* Import this file - const sendEmail = require('./email-sender);         *
+* send email - sendEmail.sendMessage("Put your error message in here");  *
+**************************************************************************/
+
+exports.sendMessage = async function (err) {
+
+    //Recipients list of users to send the email too
+    var receipiants = 'rwilson31@myseneca.ca,david.humphrey@senecacollege.ca';
+
+    //Credientials to send an email from 
+    var transporter = nodemailer.createTransport({
+        service: 'gmail',
+        auth: {
+            user: 'seneca.telescope@gmail.com',
+            pass: 'Password12345%^'
```

Comment:
This is a no-go.  Username/Password pairs need to get read in via environment variables on `process.env`

Reply:
Okay thank you, Sorry. It was a test email, I have deleted the account. 
Will change the code ASAP.